src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -387,6 +387,9 @@
         - name: --node-image-only
           type: bool
           short-summary: Only upgrade node image for agent pools.
+        - name: --aks-custom-headers
```

Comment:
how about just `--custom-headers`? the command group name is aks already, aks is redundant

Reply:
just to keep aligned with the `aks-custom-headers` in other commands like `az aks create`